staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -6066,6 +6066,100 @@ func TestStrategicMergePatch(t *testing.T) {
 	}
 }
 
+func sortJson(t *testing.B, j []byte, description string, schema LookupPatchMeta) []byte {
```

Comment:
@mengqiy Since I think you've written a lot of these test-cases, would you mind looking at this?